lib/wp2middleman/frontmatter.rb

```diff
@@ -10,7 +10,7 @@ def initialize(post, include_fields: [])
     def post_data
       data = {
         'title' => post.title,
-        'date' => post.date_published,
+        'date' => post.date_time_published,
```

Comment:
In my mind, our questions surrounding `#valid?`'s implementation can be hashed out outside this PR. I can document this effort in a GH issue.

However, for the purposes of merging this PR, I'd argue that `#valid?` -- in its current implementation -- should use `#date_time_published` rather than `#date_published`.

If you agree, would you mind editing accordingly?